### PR TITLE
added .travis.yml

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+install: true
+script: true
+before_deploy: rm -rf gh-pages && mkdir gh-pages && cp script.js style.scss gh-pages
```
Comment (dmalan) [inline .travis.yml]: These aren't the names of the files. :p

Comment (kzidane) [reply .travis.yml]: I figured `scripts` and `styles` are too many. That was a feature!

Comment (kzidane) [reply .travis.yml]: Less optimal approach: 15bd3b6.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+install: true
+script: true
+before_deploy: rm -rf gh-pages && mkdir gh-pages && cp scripts.js styles.scss gh-pages
```
Comment (dmalan) [inline .travis.yml]: `styles.css`

Please be more careful!

**Pull request merged**